### ISSUE TITLE
App stopped working after system update

### ISSUE THREAD

**Issue by Mygod** [realm01/DHCPv6-Client-Android#6]
I just updated to 7.1.2 and I can't get this to work any more.

**Comment by Mygod** [comment]
```
Apr/29/2017 11:38:23: get_duid: extracted an existing DUID from /data/misc/dhcp/dhcp6c_duid: ......
Apr/29/2017 11:38:23: dhcp6_ctl_init: bind(control sock): Address already in use
Apr/29/2017 11:38:23: client6_init: failed to initialize control channel
```

**Comment by Mygod** [comment]
Okay I removed `/data/misc/dhcp/dhcp6c_duid` and everything works again. I suspect it's a bug in my upstream router.

**Comment by realmar** [comment]
Hi @Mygod, removing this file will force the v6 client to generate a new DUID. (not recommended as per #5) But I'm happy that it solved your problem. If you need to delete this file on a regular basis, we need to think about another solution.

**Comment by Mygod** [comment]
Okay this has just happened once again after I installed another system update.

**Comment by realmar** [comment]
The last time it didn't work, was that also after a system update?

Please specify which ROM you are using and from which to which version of this ROM you updated. Also was it a version bump in the Android version or just in the ROM version?

What is the output of following commands: (run as root)
```sh
kill -9 `pgrep dhcp6c`
dhcp6c <enter-wifi-interface-name>
```

**Comment by Mygod** [comment]
Yes. This is my ROM: https://forum.xda-developers.com/oneplus-one/development/cm14-1-legendrom-t3510085

This time it's ust a version bump in the Android version (also some security patches are merged).

**Comment by realmar** [comment]
I know that I had this problem at the beginning when I started implementing the app. Therefore I decided to just delete `/data/misc/dhcp/dhcp6c_duid`. Although this is not very desirable. I need to find another solution. (See also #8)
